.travis.yml

```diff
@@ -27,18 +19,44 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
+        - php: 7.1
+          env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="composer test-ci"
+
+          # Test the latest stable release
+        - php: 5.6
+        - php: 7.0
+        - php: 7.1
+        - php: 7.2
+
+          # Test LTS versions
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci"
+          env: DEPENDENCIES="symfony/lts:^2"
+        - php: 5.5
+          env: DEPENDENCIES="symfony/lts:^3"
+
+          # Latest dev release
+        - php: 7.1
+          env: DEPENDENCIES="dev"
 
+    allow_failures:
+          # Latest beta is allowed to fail.
+        - php: 7.1
```

Comment:
no need to specify the PHP version here. It will avoid having to keep it in sync when changing the version used by `DEPENDENCIES="dev"`